mmv1/products/dataform/WorkflowConfig.yaml

```diff
@@ -35,6 +35,7 @@ examples:
     vars:
       workflow_name: 'my_workflow'
       release_name: 'my_release'
+      service_account_name: 'dataform-workflow-sa'
```

Comment:
name ends up being too long:

```
Error: "account_id" ("tf-test-dataform-workflow-sax7h3nowt7m") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"
```

Reply:
I've shortened the seed string so the generated id is short enough